proxy/core/src/main/scala/io/cloudstate/proxy/eventsourced/EventSourcedEntity.scala

```diff
@@ -179,6 +193,7 @@ final class EventSourcedEntity(configuration: EventSourcedEntity.Configuration,
 
   private[this] final def commandHandled(): Unit = {
     currentCommand = null
+    instrumentation.commandProcessed()
```

Comment:
Not sure if it matters, but shouldn't "processed" (past tense) be called at the end of the code block which does the processing? Similar applies to `commandStashed` etc. below.

Reply:
This is currently after the command has been fully processed asynchronously, including persisting events. It's in the `commandHandled` method, after the current command has been cleared and before any stashed commands are processed. So you're suggesting having a counter after it sends the command to the user function (at the end of `handleCommand`)? Or should we change the verb for this hook if `process` doesn't work? We often used `completed` for this kind of event in Cinnamon.

I'll update the stashed hook to be a line down, after the stash, and also add an unstashed hook — so we can see the number of queued commands (and this is separate from the actor stash used for persistent actors during recovery or persisting).